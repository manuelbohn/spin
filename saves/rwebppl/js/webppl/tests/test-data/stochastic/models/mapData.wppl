var model = function() {
  return mapData({data: [true, false]}, function(y) {
    var x = flip();
    factor(x === y ? 0 : -2);
    return x;
  });
};
