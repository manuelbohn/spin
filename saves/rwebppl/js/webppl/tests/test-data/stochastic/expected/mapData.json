{
  "hist": {
    "[false,false]": 0.1049936,
    "[false,true]":  0.0142093,
    "[true,false]":  0.7758035,
    "[true,true]":   0.1049936
  }
}
